### PR TITLE
ENDOC-693-bundle-tags draft

### PR DESCRIPTION
**Pull request by jyunmitch**

### CODE REVIEW

File: vuepress/docs/next/docs/compose/ecm-overview.md [hunk 1]
```diff
@@ -24,8 +24,13 @@ A similar process occurs when bundles are installed or uninstalled. The [Entando
...
+* Manages the installation and removal of bundles
 
-* Makes components available in the App Builder through the Local Hub
+* Makes bundles available in the App Builder through the Local Hub
 
-* Handles versioning of component bundles for sharing and collaborating
+* Offers the option to install bundles for development, production or both using Docker tags: 
+    * The environment variable `ENTANDO_BUNDLE_TAGS_TYPES` in the Entando Component Manager can be used to select for development (`dev`), production (`prod`), or both (`prod,dev`) type of bundles to be installed to the Local Hub. The default is to install bundles with production tags only.  
+    
+    * To specify what is installed for each individual bundle, see the [Bundle Management page](../getting-started/ent-bundle.md#generate-cr) to generate the bundle custom resouce with the desired tag type.
```
Comment (firegloves) [inline vuepress/docs/next/docs/compose/ecm-overview.md:34]: is `resouce` a typo?

File: vuepress/docs/next/docs/getting-started/ent-bundle.md [hunk 2]
```diff
@@ -152,25 +152,36 @@ See the [Build and Publish a Simple Bundle](../../tutorials/create/pb/publish-si
...
 |`ent bundle install`| Install a bundle in the Local Hub of an Entando Application |
 
-#### Command Details
+#### Install Command Details
 * `ent bundle install --conflict-strategy=OVERRIDE`: If a bundle project has already been installed, the `--conflict-strategy` flag forces a `CREATE`, `SKIP` or `OVERRIDE` strategy for components
 
+### Generate CR
+| Command| Description
+|:--|:--
+|`ent bundle generate-cr -t []`| Generate the Custom Resource (CR) for a bundle project, including versions based on tags for development, production or both. The default is the  production tag type.
```
Comment (firegloves) [inline vuepress/docs/next/docs/getting-started/ent-bundle.md:161]: There is a double space between `the  production`

**Review by firegloves — APPROVED** [review]
LGTM

File: vuepress/docs/next/docs/compose/ecm-overview.md
Comment (nshaw) [inline vuepress/docs/next/docs/compose/ecm-overview.md:1]: I think this feature was completed with 7.2 so it should also be applied to the 7.2 docs.

**Pull request merged**